examples/psp22-extension/lib.rs

```diff
@@ -0,0 +1,267 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+#![allow(clippy::new_without_default)]
```

Comment:
Can you address this lint?

Reply:
I though it cannot be done, because of `no_std`. I see that the same `allow` annotation is added in four other examples in this repo.